Fix bug header version

shimastripe opened the pull request:
# WHY

- found a bug where I have implemented 
- when get the version from Accept-Header.

# WHAT

- fix bug

dtan4 commented:
@shimastripe Could you show me the sample header?

I'm unsure whether is it okay to separate Accept header string by semicolon... It seems that [RFC 2616](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html) indicates the separator is comma.

shimastripe commented:
@dtan4 
First, I think this way.

```
Accept: application/vnd.wantedly+json; version=1.0.0
```

Then I write a code as follows.

```
ver = strings.Split(header, "=")[1]
```

But  "=", it is also used such as in the case of a charset.

```
Accept: application/vnd.wantedly+json; charset=utf-8; version=1.0.0
```

So I improve the code as follows.

```
ver = strings.Split(strings.SplitAfter(header, "version=")[1], ";")[0]
```

The separator of the accept-header's list seems to be only ";".

dtan4 commented:
I got it. LGTM 👍 Thanks ❗
